### PR TITLE
[ADD][9.0] module 'account_move_line_purchase_info'

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Account Move Line Purchase Info
===============================

This module will add the purchase order line to journal items.

The ultimate goal is to establish the purchase order line as one of the key
fields to reconcile the Goods Received Not Invoiced accrual account.

Usage
=====

The purchase order line will be automatically copied to the journal items.

* When a supplier invoice is created referencing purchase orders, the
  purchase order line will be copied to the corresponding journal item.

* When a stock move is validated and generates an journal entry, the purchase
 order line is copied to the account move line.

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@mreficent @gmeficent can you review?

File: account_move_line_purchase_info/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/154/9.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/purchase-workflow/issues>`_. In case of trouble, please
```
Comment (MiquelRForgeFlow) [inline account_move_line_purchase_info/README.rst]: purchase-workflow → account-financial-tools

File: account_move_line_purchase_info/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+{
+    "name": "Account Move Line Purchase Info",
+    "summary": "Introduces the purchase order line to the journal items",
+    "version": "9.0.1.0.0",
+    "author": "Eficent, "
+              "Odoo Community Association (OCA)",
+    "website": "http://www.eficent.com",
+    "category": "Generic",
+    "depends": ["account", "purchase"],
```
Comment (MiquelRForgeFlow) [inline account_move_line_purchase_info/__openerp__.py]: account → accounting

File: account_move_line_purchase_info/README.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+Usage
+=====
+
+The purchase order line will be automatically copied to the journal items.
+
+* When a supplier invoice is created referencing purchase orders, the
+  purchase order line will be copied to the corresponding journal item.
+
+* When a stock move is validated and generates an journal entry, the purchase
```
Comment (MiquelRForgeFlow) [inline account_move_line_purchase_info/README.rst]: an journal → a journal

**Review by MiquelRForgeFlow — APPROVED** [review]
tested on local :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
Functional and technical review :+1:

**Review by gmeficent — APPROVED** [review]
technical review and tested in runbot :+1:

**Pull request merged**